frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -71,13 +93,20 @@ function KeyValuePairDisplay({ left, right }: { left: string; right: any }) {
             >
                 {right}
             </Typography>
-        </>
+        </StyledFormItem>
     )
 }
 
-function MissionDefinitionPageBody({ missionDefinition, updateMissionDefinition }: IProps) {
+interface IMissionDefinitionPageBodyProps {
+    missionDefinition: CondensedMissionDefinition
+    updateMissionDefinition: (missionDefinition: CondensedMissionDefinition) => void
+}
+
+function MissionDefinitionPageBody({ missionDefinition, updateMissionDefinition }: IMissionDefinitionPageBodyProps) {
     const { TranslateText } = useLanguageContext()
     let navigate = useNavigate()
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState<boolean>(false)
+    const [selectedField, setSelectedField] = useState<string>('')
```

Comment:
On line 111 - 123:
Do we need minutes in line 116?

Also suggest removing hour cause I think we can assume that inspection frequency will be in days (can add it again if it turns out to be wrong)

Reply:
Suggest that it says "Inspeksjon kreves hver 4. dag" instead
![image](https://github.com/equinor/flotilla/assets/33518988/34d6c991-7dd3-4568-815f-b05ef34a96d2)